### PR TITLE
(DOCSP-14917): Local tested examples of Realm Objects

### PR DESCRIPTION
**Pull request by nathan-contino-mongo**
### Issue JIRA link:
https://jira.mongodb.org/browse/DOCSP-14917

NOTE: This PR contains code snippets generated by [Bluehawk](https://github.com/mongodb-university/Bluehawk). As a result, there are a *lot* of lines changed, files created/deleted -- had to do a bit of housekeeping with the generated snippets here. However, the reviewer should just be able to take a look at the staging link pages for the actual review:

### Docs staging link (requires sign-in on MongoDB Corp SSO):
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14917/sdk/android/examples/define-a-realm-object-model/ (code snippets and moderate amount of copy)
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14917/sdk/android/examples/open-and-close-a-local-realm/ (code snippets only)
https://docs-mongodbcom-staging.corp.mongodb.com/realm/docsworker-xlarge/DOCSP-14917/sdk/android/quick-start-local/ (code snippets and small amount of copy)

### Docs staging build link:
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?jobId=603d8f5975e817aef705c7a1

### Review Guidelines
[REVIEWING.md](https://github.com/mongodb/docs-realm/blob/master/REVIEWING.md)

### CODE REVIEW

File: source/sdk/android/examples/define-a-realm-object-model.txt [hunk 1]
```diff
@@ -86,19 +113,21 @@ whose type is a {+service-short+} object in your application:
...
+      .. include:: /examples/generated/android/local/FrogManyToOneExampleKt.codeblock.complete.kt.code-block.rst
 
    .. tab::
       :tabid: java
 
-      .. literalinclude:: /examples/Schemas/ManyToOne.java
-         :language: java
+      .. include:: /examples/generated/android/local/FrogManyToOneExample.codeblock.complete.java.code-block.rst
+
+Each ``Frog`` references zero or one other ``Frog`` instances. Nothing
```
Comment (dacharyc) [inline source/sdk/android/examples/define-a-realm-object-model.txt:123]: Nit: "instances" in this line reads weird to me. If you separate these, you have "references zero instances" or "references one other instance."

**Review by dacharyc — APPROVED** [review]
One minor nit but LGTM! And thanks for the helpful info about what to actually review - that is a lot of files!

File: source/sdk/android/examples/define-a-realm-object-model.txt (hunk 1 above)
Comment (nathan-contino-mongo) [reply source/sdk/android/examples/define-a-realm-object-model.txt:123]: How about "zero or more"?

Comment (dacharyc) [reply source/sdk/android/examples/define-a-realm-object-model.txt:123]: That works!

Comment (nathan-contino-mongo) [inline source/sdk/android/examples/define-a-realm-object-model.txt:123]:
```suggestion
Each ``Frog`` references zero or more other ``Frog`` instances. Nothing
```

File: source/sdk/android/examples/define-a-realm-object-model.txt [hunk 2]
```diff
@@ -86,19 +113,21 @@ whose type is a {+service-short+} object in your application:
...
+      .. include:: /examples/generated/android/local/FrogManyToOneExampleKt.codeblock.complete.kt.code-block.rst
 
    .. tab::
       :tabid: java
 
-      .. literalinclude:: /examples/Schemas/ManyToOne.java
-         :language: java
+      .. include:: /examples/generated/android/local/FrogManyToOneExample.codeblock.complete.java.code-block.rst
+
+Each ``Frog`` references zero or more other ``Frog`` instances. Nothing
```
Comment (nathan-contino-mongo) [inline source/sdk/android/examples/define-a-realm-object-model.txt:123]:
```suggestion
Each ``Frog`` references either zero ``Frog`` instances or one other ``Frog`` instance. Nothing
```

**Pull request merged**